python/nano/src/bigdl/nano/deps/openvino/tf/model.py

```diff
@@ -39,11 +40,14 @@ def __init__(self, model):
             if isinstance(model, tf.keras.Model):
                 export(model, str(dir / 'tmp.xml'))
                 ov_model_path = dir / 'tmp.xml'
-            OpenVINOModel.__init__(self, ov_model_path)
+            self.ov_model = OpenVINOModel(ov_model_path)
             AcceleratedKerasModel.__init__(self, None)
 
+    def forward_step(self, *inputs):
+        return self.ov_model.forward_step(*inputs)
+
     def on_forward_start(self, inputs):
-        if self.ie_network is None:
+        if self.ov_model.ie_network is None:
```

Comment:
can you move this check to `OpenVINOModel`? I think this check should be the same for Pytorch/TF/OpenVINO.

Reply:
should the error message be different? or just keep the same.